testgrid/config.yaml

```diff
@@ -6728,6 +6728,8 @@ dashboard_groups:
   - presubmits-poseidon
   - presubmits-kube-batch
   - presubmits-misc
+  - presubmits-release-notes-master
```

Comment:
FYI @saschagrunert (ref: https://github.com/kubernetes/test-infra/pull/12656)

Reply:
Ah okay thanks for letting me know :)